src/ansys/dpf/core/operators/result/__init__.py

```diff
@@ -59,7 +59,6 @@
 from .coordinates import coordinates
 from .creep_strain_energy_density import creep_strain_energy_density
 from .current_density import current_density
-from .custom import custom
```

Comment:
Although you have recorder `"custom"` in the server, we need it to be public to generate its API, as not having it here means that `dpf.operators.result.custom` will fail in any production script and all users would need to update the scripts with the same change you have done to the tests, which is not good